test/vpc_test.go

```diff
@@ -0,0 +1,31 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+)
+
+func TestAccIBMISVPC(t *testing.T) {
+	t.Parallel()
+
+	// Construct the terraform options with default retryable errors to handle the most common retryable errors in
+	// terraform testing.
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		// The path to where our Terraform code is located
+		TerraformDir: ".",
```

Comment:
in current directory we don't have any templates to execute ... I think it should be pointing to example directory